lib/Dancer2/Core/App.pm

```diff
@@ -833,6 +833,41 @@ sub log {
     $logger->$level(@_);
 }
 
+sub send_as {
+    my $self = shift;
+    my ( $type, $data, $options ) = @_;
+    $options ||= {};
+
+    # allow lower cased serializer names
+    my $prefix = 'Dancer2::Serializer::';
+    my $serializer_class;
+    for ( uc $type, $type ) {
+        if ( Class::Load::try_load_class( "$prefix$_" ) ) {
```

Comment:
I'm convinced. Shall we open a ticket for deciding on replacing Class::Load with Module::Runtime? We could add a benchmark, have a quick convo, decide, and then either do it or close the ticket.